src/ssl.c

```diff
@@ -87,6 +87,10 @@
     #ifdef WOLFSSL_SHA512
         #include <wolfssl/wolfcrypt/sha512.h>
     #endif
+    #if defined(WOLFCRYPT_HAVE_SRP) && !defined(NO_SHA256) && !defined(WC_NO_RNG)
```

Comment:
These lines is embraced by outer #ifdef OPENSSL_EXTRA.
So I don't need to check in this line.